ocaml/vhd-tool/src/cohttp_unbuffered_io.ml

```diff
@@ -100,7 +100,7 @@ let rec read_line ic =
       read_line ic
   | Some buf, i when i < String.length buf -> (
     try
-      let eol = Re.Str.search_forward crlf buf i in
+      let eol = Re.Group.start (Re.exec ~pos:i crlf buf) 0 in
```

Comment:
I don't think we necessarily need a regex here, this just looks for the `\r\n` substring, no alternatives are usd. Would something like Astring's String.find_sub work here?